qabelbox/src/main/java/de/qabel/qabelbox/providers/DocumentIdParser.java

```diff
@@ -21,7 +21,7 @@ public String getIdentity(String documentId) throws FileNotFoundException {
 
 
     public String getPrefix(String documentId) throws FileNotFoundException {
-        String[] split = documentId.split(BoxProvider.DOCID_SEPARATOR, 4);
+        String[] split = documentId.split(BoxProvider.DOCID_SEPARATOR, 3);
```

Comment:
would you mind extracting the magic number to something like MAX_ID_ELEMENT_COUNT ?